### PR TITLE
Add theme & lib/toast UI exports

### PR DESCRIPTION
**Pull request by JasonMHasperhoven**
Currently we can't access theme and lib/toast within the dex-explorer. This should fix that.

### CODE REVIEW

**Review by TalDerei — APPROVED** [review]

File: packages/ui/package.json [hunk 1]
```diff
@@ -46,6 +48,18 @@
         "types": "./dist/components/ui/*.d.ts",
         "default": "./dist/components/ui/*.js"
       },
+      "./theme": {
+        "types": "./dist/src/PenumbraUIProvider/theme.d.ts",
+        "default": "./dist/src/PenumbraUIProvider/theme.js"
+      },
+      "./lib/toast/*": {
```
Comment (VanishMax) [inline packages/ui/package.json:55]: issue: this is the old toast library, it should not be exported. The v2 one is already working in the dex: https://github.com/penumbra-zone/dex-explorer/pull/118/files#diff-e1b5cf1eb29b361c28415989517c8312f1f349ab3e0cac44425ffc02290259c2R62

File: packages/ui/package.json [hunk 2]
```diff
@@ -46,6 +48,18 @@
         "types": "./dist/components/ui/*.d.ts",
         "default": "./dist/components/ui/*.js"
       },
+      "./theme": {
```
Comment (VanishMax) [inline packages/ui/package.json:51]: question: why is the theme needed to be exported, especially this way? We use `PenumbraUIProvider` in the `app/app.tsx` in the DEX.

suggestion: let's just export the `theme` object from `PenumbraUIProvider/index.tsx` file

File: packages/ui/package.json [hunk 3]
```diff
@@ -33,6 +33,8 @@
     "./tailwind": "./src/tailwindConfig.ts",
     "./postcss.config.js": "./postcss.config.js",
     "./styles/*": "./styles/*",
+    "./theme": "./src/PenumbraUIProvider/theme.ts",
+    "./utils/*": "./src/utils/*.ts",
```
Comment (VanishMax) [inline packages/ui/package.json:37]: question: is there something in the v1 utils we need in the DEX?

**Review by VanishMax — COMMENTED** [review]
If possible, I believe this PR should be re-done

**Pull request merged**